tdh/utils/task.go

```diff
@@ -16,8 +16,8 @@ func WaitForTask(client *tdh.Client, taskId string) error {
 		}
 		if taskResponse.Status == "SUCCESS" {
 			return nil
-		} else if taskResponse.Status == "FAILED" {
-			return fmt.Errorf("task has failed, get more details using datasource \"tdh_tasks\"")
+		} else if taskResponse.Status == "FAILED" || taskResponse.Status == "RESCHEDULED" {
```

Comment:
remove rescheduled condition, many tasks happen to get SUCCESS later on retry, RESCHEDULED does not mean fail, it may or may not fail